ddp-workspace/package.json

```diff
@@ -32,7 +32,7 @@
     "@ngx-translate/core": "13.0.0",
     "@ngx-translate/http-loader": "6.0.0",
     "auth0-js": "9.14.0",
-    "auth0-lock": "11.27.0",
```

Comment:
Are these changes required for the dashboard to render properly?  If not, please remove them--we try to avoid auto-upgrade (`^`) in our package.json.